ltr/concourse/deploy.py

```diff
@@ -40,13 +42,16 @@
     print("could not find current endpoint, will create a new one", file=sys.stderr)
 
 # find the model
+model_location = model_tag or model_name
 model_keys = session.boto_session.client("s3").list_objects_v2(
-    Bucket=s3_bucket, Prefix=f"model/{model_name}"
+    Bucket=s3_bucket, Prefix=f"model/{model_location}"
 )["Contents"]
 if len(model_keys) != 1:
     raise Exception("Expected exactly 1 model key")
 model_key = model_keys[0]["Key"]
 
+print(f"Deploying model {model_key} (found at model/{model_location}) to Sagemaker...", file=sys.stderr)
```

Comment:
`{model_key}` will actually have the full s3 path of the thing

Reply:
Ah cool, I don't need to set `model_location` :)